### PR TITLE
Adjustments after call with Simon

### PR DESCRIPTION
**Pull request by dexter2206**
## Description

This PR adjusts several formulas and instroduces names for some intermediate variables to hopefully make our `GraphResourceEstimator` code more readable.

## Please verify that you have completed the following steps

- [x] I have self-reviewed my code.
- [x] I have included test cases validating introduced feature/fix.
- [x] I have updated documentation.

### CODE REVIEW

File: src/benchq/resource_estimation/graph/estimators.py [hunk 1]
```diff
@@ -65,32 +68,41 @@ def _logical_cell_failure_rate(self, distance: int) -> float:
...
         _, ec_error_rate = self.balance_logical_error_rate_and_synthesis_accuracy(
-            n_nodes, distance
+            n_nodes, distance, max_graph_degree
         )
         return ec_error_rate
 
-    def get_logical_st_volume(self, n_operations):
-        return 12 * n_operations * self.N_TOCKS_PER_T_GATE_FACTORY * n_operations
+    def get_logical_st_volume(self, n_nodes: int, max_graph_degree: int):
+        num_boxes = np.ceil((max_graph_degree + 1) / 21)
```
Comment (mstechly) [inline src/benchq/resource_estimation/graph/estimators.py:101]: If this is the same 21 as previously it should be in some variable.

File: src/benchq/resource_estimation/graph/estimators.py [hunk 2]
```diff
@@ -65,32 +68,41 @@ def _logical_cell_failure_rate(self, distance: int) -> float:
...
+        space = 21 * 12 * num_boxes
```
Comment (mstechly) [inline src/benchq/resource_estimation/graph/estimators.py:102]: Same for 12 here.

File: src/benchq/resource_estimation/graph/estimators.py [hunk 3]
```diff
@@ -129,48 +142,50 @@ def balance_logical_error_rate_and_synthesis_accuracy(self, n_nodes, distance):
...
             6 * self.hw_model.physical_gate_time_in_seconds * code_distance
         )
 
         wall_time = (
-            time_of_logical_t_gate
+            n_measurement_steps * time_of_logical_t_gate
+            + time_of_logical_t_gate
             * self.N_TOCKS_PER_T_GATE_FACTORY
-            * n_measurement_steps
+            * n_nodes
```
Comment (mstechly) [inline src/benchq/resource_estimation/graph/estimators.py:183]: Curious where does this come from.

File: src/benchq/resource_estimation/graph/estimators.py (hunk 2 above)
Comment (AthenaCaesura) [reply src/benchq/resource_estimation/graph/estimators.py:102]: Yeah, it could be named something like `T_GATE_FACTORY_WITDH`

**Review by mstechly — APPROVED** [review]

**Review by AthenaCaesura — APPROVED** [review]

**Pull request merged**